components/support/versioning/src/error.rs

```diff
@@ -0,0 +1,24 @@
+use std::fmt;
+
+/// The `VersioningError` indicates that a Firefox Version, being passed down as a `String`,
+/// cannot be parsed into a `Version`.
+///
+/// It can either be caused by a non-ASCII character or a integer overflow.
+#[derive(Debug, PartialEq)]
+pub enum VersioningError {
```

Comment:
The old enum variant was VersionParsingError. Lets keep that name, so its very clear what the error is about.